classes/form/CustomerAddressFormatter.php

```diff
@@ -97,6 +97,9 @@ public function getFormat()
                         $formField->setRequired(true);
                     }
                 }
+                if ($field === 'phone') {
```

Comment:
An `elseif` or a `switch` statement could be better 👍

Reply:
Hi @PierreRambaud
You are right.
`elseif` is better, but I don't see the need to use o a `switch` statement for just 2 cases.
I made another commit.